packages/fxa-shared/lib/errors.ts

```diff
@@ -133,6 +133,8 @@ export const AUTH_SERVER_ERRNOS = {
 
   SMS_SEND_RATE_LIMIT_EXCEEDED: 216,
 
+  RECOVERY_PHONE_REMOVE_MISSING_RECOVERY_CODES: 217,
```

Comment:
Looks like this error number might conflict with another error that Barry recently added and merged